src/python/nimbusml/pipeline.py

```diff
@@ -1771,6 +1773,13 @@ def get_feature_contributions(self, X, top=10, bottom=10, verbose=0,
             data="$data",
             predictor_model="$predictor_model",
             scored_data="$scoredvectordata")
+        
+        if isinstance(X, csr_matrix):
```

Comment:
Should there be a test which verifies that `get_feature_contributions` works with csr matrices?